# requested command
HybridImpedanceSwitchForcePosMsg command
---
# current command
HybridImpedanceSwitchForcePosMsg command
